pkgs/desktops/gnome-3/extensions/update-extensions.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+import urllib.request, urllib.error, json, types
+from typing import List, Set, Dict, Optional
+
+# We don't want all those deprecated legacy extensions
+# Group extensions by Gnome version for compatibility reasons
+supported_versions = {
+    "36": {
```

Comment:
Can we generate this information somehow? This will likely bitrot.

Reply:
I will try to modify it so that one only has to specify major versions.